modular_newhorizons/master_files/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -0,0 +1,24 @@
+/singleton/reagent/toxin/potassium_chlorophoride
+	description = "A specific chemical based on Potassium Chloride to stop the heart for surgery. Not safe to eat!"
+
+/singleton/reagent/lexorin
+	description = "Lexorin temporarily stops respiration. Causes tissue damage."
+
+/singleton/reagent/soporific
+	description = "An effective hypnotic used to treat insomnia, can act as a sedative. Lasts three times longer when inhaled."
+
+/singleton/reagent/polysomnine
+	name = "Chloral Hydrate"
+	description = "A powerful sedative. Lasts two times longer when inhaled."
+
+/singleton/reagent/slimetoxin
+	description = "A corruptive toxin produced by slimes."
+
+/singleton/reagent/aslimetoxin
+	description = "An advanced corruptive toxin produced by slimes."
+
+/singleton/reagent/toxin/berserk
+	description = "An illegal chemical enhancer, may cause aggressive and violent behavior."
```

Comment:
" An illegal combat performance enhancer originating from the criminal syndicates of Mars. The drug stimulates regions of the brain responsible for violence and rage, inducing a feral, berserk state in users." seems fine I would say, same list as the slime toxins with no override.